packages/koa-shopify-webhooks/CHANGELOG.md

```diff
@@ -5,8 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.1]
```

Comment:
This can go under an `## Unreleased` header.  This way, maintainers can examine all recent changes, and make an informed decision about it being a major/minor/patch version.

(It's almost definitely going to be a patch version, but I have to say this anyway 😛)